src/api/flaskr/api/tts/minimax_provider.py

```
@@ -78,6 +86,119 @@
+    trace_suffix = f", trace_id={trace_id}" if trace_id else ""
+    return f"{prefix}: {status_code} - {status_msg}{trace_suffix}"
+
+
+def _fetch_minimax_subtitle_file(url: str) -> List[Dict[str, Any]]:
+    if not url:
+        return []
+    try:
+        response = requests.get(url, timeout=20)
+        response.raise_for_status()
+        payload = response.json()
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)